f/connectors/alerts/alerts_gcs.py

```diff
@@ -96,11 +101,14 @@ def _main(
 
     convert_tiffs_to_jpg(tiff_files)
 
+    alerts_metadata_table_name = f"{db_table_name}__metadata"
     prepared_alerts_metadata, alerts_statistics = prepare_alerts_metadata(
-        alerts_metadata, territory_id
+        alerts_metadata, territory_id, alerts_provider, db, alerts_metadata_table_name
     )
 
-    prepared_alerts_data = prepare_alerts_data(destination_path, geojson_files)
+    prepared_alerts_data = prepare_alerts_data(
+        destination_path, geojson_files, alerts_provider
+    )
```

Comment:
I don't recall the distinction between the `alerts` vs `alerts__metadata` tables... but it appears that `data_source` is a column in both of them. So why does `prepare_alerts_data` not also have to do the same disambiguation of which rows are already in this table vs new (using `_get_existing_data_sources`)?